Header header
uint8 type
uint8 compressed
uint8 compressed_level
float32 lsb
uint8[] hdf5data

uint8 TYPE_XYZI=0
uint8 TYPE_XYZRGB=1